examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.current = 0
+
+    this.context.questions_and_answers =
+      [
+        {
+          question: 'what popular tool for developing the client side application should we use?',
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'Bootstrap', 'jQuery'],
+          // answer: undefined // There is no answer hehe
+        },
+
+        {
+          question: 'What city is known as "The Big 🍎 Apple ?"',
+          answers: ['Seattle, WA', 'Skokie, IL', 'New York City'],
+          answer: 2
+        }
+      ]
+  }
+
+  onidle () {
+    const
```

Comment:
This style looks VERY peculiar to me. Why isn't `footer button...` etc) on the same line? It'd be only 58 charr long (for the first one)
Also, leading commas, not trailing?